maps_dashboards/public/components/layer_config/document_layer_config_panel.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { Fragment } from 'react';
+import { EuiSpacer, EuiText, EuiTabbedContent } from '@elastic/eui';
+import { ILayerConfig } from '../../model/ILayerConfig';
+import { LayerBasicSettings } from './layer_basic_settings';
+import { DocumentLayerSource } from './document_layer_source';
+import { DocumentLayerStyle } from './document_layer_style';
+
+interface Props {
+  selectedLayerConfig: ILayerConfig;
+  setSelectedLayerConfig: Function;
+}
+
+export const DocumentLayerConfigPanel = ({
+  selectedLayerConfig,
+  setSelectedLayerConfig,
+}: Props) => {
+  const tabs = [
+    {
+      id: 'data-source--id',
+      name: 'Data source',
```

Comment:
nit: Cant we simply call this Data?

Reply:
Updated, one word looks cleaner. We can check on the UI. cc: @VijayanB @btzeng